Woof/Woof/View/SitterFlow/EditSitterInformationView.swift

```diff
@@ -0,0 +1,106 @@
+import SwiftUI
+
+/// A view to edit the sitter information.
+struct EditSitterInformationView: View {
+    /// The name of the sitter.
+    @Binding var name: String
+
+    /// The surname of the sitter.
+    @Binding var surname: String
+
+    /// The phone of the sitter.
+    @Binding var phone: String
+
+    /// The additional information about the sitter, like his experience with dogs, favourite places for walks,
+    /// special skills, certificates, etc.
+    @Binding var bio: String
+
+    /// The price per hour for walking charged by the sitter.
+    @Binding var pricePerHour: String
+
+    var body: some View {
+        VStack {
+            Text("Edit your information")
+            TextField("Enter your name", text: $name)
```

Comment:
Please extract strings into the private interface:

```suggestion
            TextField(namePlaceholderText, text: $name)
            
            private let namePlaceholderText = "Enter your name"
```

Reply:
Thanks, fixed it.